dirty_cat/_fuzzy_join.py

```diff
@@ -61,8 +67,8 @@ def fuzzy_join(
         Must be found in both DataFrames. Use only if `left_on`
         and `right_on` parameters are not specified.
     analyzer : typing.Literal["word", "char", "char_wb"], default=`char_wb`
-        Analyzer parameter for the CountVectorizer used for the string
-        similarities.
+        Analyzer parameter for the CountVectorizer/HashingVectorizer used for
+        the string similarities.
```

Comment:
```suggestion
        Analyzer parameter for the CountVectorizer/HashingVectorizer
        (passed by `encoder`) used for the string similarities.
```

Reply:
Shouldn't it be "passed *to* encoder" instead?